.github/actions/install-psi4/action.yml

```diff
@@ -25,11 +25,21 @@ runs:
     - run : |
         source "$CONDA/etc/profile.d/conda.sh"
         conda activate psi4env
-        if [ "${{ inputs.os }}" == "windows-2019" ]; then
-          conda install -y psi4=1.5 python=${{ inputs.python-version }} -c conda-forge -c psi4/label/dev
+        if [[ "${{ inputs.os }}" == "windows-2019" ]]; then
+          release=1.5
+          echo "installs psi4 release $release"
+          conda install -y psi4=$release python=${{ inputs.python-version }} -c conda-forge -c psi4/label/dev
           psi4 --test
```

Comment:
Why is this being run on Windows?

Reply:
This has been like that since last year release 1.5. It is the required install on Windows using conda-forge.
I run a quick test afterwards to make sure all the dlls load correctly